competition/prerace_setup.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from animal_factory import factory
+from competition.Round import Round
+from config import RACERS_CONFIG_PATH
+from constants import ORIGIN0_OFFSET
+
+TRACK_LENGTH = 'track_length'
+RACERS = 'racers'
+ANIMAL_TYPE = 'type'
+RACERS_CONFIG = 'racers_config'
+RACE_SETTINGS = 'race_settings'
+ROUNDS = 'rounds'
+
+
+def initialize_rounds(race_settings: dict) -> list:
+    rounds = [
```

Comment:
you can just return this.
When it doesn't matter it's usually best practice to do a generator instead of list comprehension (just use () instead of []), because it's more memory efficient. It's quite minor in this case because rounds is very short, so that's fine as is